docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -557,6 +557,8 @@ Define allowed algorithms and their minimum key length (use -1 to disable a type
 - `COOKIE_NAME`: **i\_like\_gitea**: The name of the cookie used for the session ID.
 - `GC_INTERVAL_TIME`: **86400**: GC interval in seconds.
 - `SESSION_LIFE_TIME`: **86400**: Session life time in seconds, default is 86400 (1 day)
+- `DOMAIN`: **\<empty\>**: Sets the cookie Domain
```

Comment:
By default imho it should set http domain imho

Reply:
As per discussion on discord. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie .
Leaving this blank is actually the correct thing to do.